java/src/main/java/com/powsybl/python/PyPowsyblNetworkApiLib.java

```diff
@@ -398,6 +400,21 @@ public static void updateNetworkElementsWithSeries(IsolateThread thread, ObjectH
         });
     }
 
+    @CEntryPoint(name = "removeNetworkElements")
+    public static void removeNetworkElements(IsolateThread thread, ObjectHandle networkHandle, CCharPointerPointer cElementIds,
+                                             int elementCount, PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+        doCatch(exceptionHandlerPtr, () -> {
+            Network network = ObjectHandles.getGlobal().get(networkHandle);
+            List<String> elementIds = CTypeUtil.toStringList(cElementIds, elementCount);
+            elementIds.forEach(elementId -> {
+                List<Connectable> connectables = network.getConnectableStream()
```

Comment:
For each element ID there can be only ONE identifiable --> no need to use a list.

Actually the more efficient will be to use `network.getIdentifiable(String id)`, in order to avoid linear searches.